storio-common/src/main/java/com/pushtorefresh/storio/operations/PreparedWriteOperation.java

```diff
@@ -0,0 +1,18 @@
+package com.pushtorefresh.storio.operations;
+
+import android.support.annotation.CheckResult;
+import android.support.annotation.NonNull;
+
+import rx.Completable;
+
+/**
+ * Common API of prepared write operations
+ *
+ * @param <Result> type of result
+ */
+public interface PreparedWriteOperation<Result> extends PreparedOperation<Result> {
```

Comment:
Do you have a better name in mind? I did it because I didn't want to force implement `asRxCompletable` for read operation.